server/src/entity/Sessions.ts

```diff
@@ -0,0 +1,23 @@
+import {BaseEntity, Column, Entity, PrimaryColumn} from 'typeorm';
+import {SessionEntity} from 'typeorm-store';
+
+@Entity()
+export class Sessions extends BaseEntity implements SessionEntity {
```

Comment:
Lets name entities in singular form. They are represented single entity, not list.